macapp/App/Sources/TestSupport/TestSupport.swift

```diff
@@ -23,7 +23,7 @@ public extension UUID {
   }
 }
 
-extension UUID: ExpressibleByIntegerLiteral {
+extension UUID: @retroactive ExpressibleByIntegerLiteral {
```

Comment:
this is the only `@retroactive` where we don't own either the type or the protocol, but i'm ok because it's only in a test target, and it's so darn useful.